lib/Transforms/Scalar/UnifyAssumes.cc

```diff
@@ -81,15 +95,35 @@ bool UnifyAssumesPass::runOnFunction(Function &F) {
   auto *assumeFn = SBI.mkSeaBuiltinFn(seahorn::SeaBuiltinsOp::ASSUME, M);
 
   SmallVector<CallInst *, 16> assumes;
+  // an orphan assert is one which does not have a preceding assume
```

Comment:
I am very confused by this code. Marking "to discuss"

Reply:
removed